### DISCUSSION TITLE
Where is the swagger/OpenApi UI for dotnet-monitor?

### DISCUSSION THREAD

**Discussion by paulomorgado** [dotnet/dotnet-monitor#6666] [Q&A] [answered]
Used this be available with `mcr.microsoft.com/dotnet/monitor`

**Comment by jander-msft** [comment, marked as answer]
Per the release notes, we've removed the Swagger UI from .NET Monitor and replaced it with the OpenAPI document:
- https://github.com/dotnet/dotnet-monitor/releases/tag/v8.0.2
- https://github.com/dotnet/dotnet-monitor/releases/tag/v7.3.4

It was removed in an effort to harden the unauthenticated API surface of .NET Monitor. From the release notes, you'll see we did some additional security hardening regarding API key validation on all versions of .NET Monitor.

The Swagger UI is not considered core functionality of .NET Monitor but more of a convenience measure to allow some type of UX over the API surface. There are other tools that you can use (e.g. Postman, Insomnia, etc; note these are not endorsements but examples of such tools) that have UI for OpenAPI documents and interacting with the described routes.

**Reply by paulomorgado** [reply]
Makes sense!
